python_examples/basic_infection_history_simulation/triple_simulation_flow.py

```diff
@@ -0,0 +1,75 @@
+#
```

Comment:
Do you mean to commit this file to main? I don't think we necessarily need it there, as I am not sure that it is useful to a public audience (and was just for internal debugging). If it has wider use then we should document this in the readme.

Reply:
See prev comment (mistake)